usaspending_api/awards/management/commands/load_subawards_sql/050_enhance_with_transaction_data.sql

```diff
@@ -20,3 +20,17 @@ from
 where
     fpds.transaction_id = temp_load_subawards_subaward.latest_transaction_id and
     temp_load_subawards_subaward.latest_transaction_id is not null;
+
+
+update
+    temp_load_subawards_subaward
+
+set
+    place_of_perform_scope = fabs.place_of_performance_scope
+
+from
+    transaction_fabs fabs
+
+where
+    fabs.transaction_id = temp_load_subawards_subaward.latest_transaction_id and
+    temp_load_subawards_subaward.latest_transaction_id is not null;
```

Comment:
I would suggest putting this query in its own file.  If memory serves, Postgres only returns the row count of the last query executed.  If we want the row count from the query above logged, these two queries will need to be in their own files.

Also, `and temp_load_subawards_subaward.latest_transaction_id is not null` is not strictly necessary.  In SQL NULL != NULL so the first part of the _where_ clause should never match if either side of the `=` is NULL.  I don't think affects anything to leave it so not a showstopper.

Reply:
Should I remove the extraneous check from the original SQL as well?

Reply:
Might as well.  We'll have to track down the original developer and flog them for such nonsense.